test/linalg/bidiag.jl

```diff
@@ -97,29 +97,30 @@ srand(1)
         end
 
         @testset "triu and tril" begin
+            bidiagcopy(dv, ev, uplo) = Bidiagonal(copy(dv), copy(ev), uplo)
```

Comment:
e0d75d7e998cf04e6634f4b92c6d5dbc2776dcb5 only changed the behaviour of `Bidiagonal`. The other ones still have constructors that don't copy: [Diagonal](https://github.com/JuliaLang/julia/blob/72873219815efdc25cc71bb6d667f2db14f117a1/base/linalg/diagonal.jl#L48-L49), [Tridiagonal](https://github.com/JuliaLang/julia/blob/72873219815efdc25cc71bb6d667f2db14f117a1/base/linalg/tridiag.jl#L432) and [SymTridiagonal](https://github.com/JuliaLang/julia/blob/72873219815efdc25cc71bb6d667f2db14f117a1/base/linalg/tridiag.jl#L48).